diff -ruN orig/rpm/tvheadend.sysconfig new/rpm/tvheadend.sysconfig
--- a/rpm/tvheadend.sysconfig	2018-10-08 09:26:34.000000000 -0600
+++ b/rpm/tvheadend.sysconfig	2018-10-23 13:46:49.235032507 -0600
@@ -1,4 +1,4 @@
 # Configuration file for the tvheadend service.
 
 MALLOC_ARENA_MAX=4
-OPTIONS="-c /var/lib/tvheadend/config -u tvheadend -g tvheadend -6 --http_port 9981 --htsp_port 9982"
+OPTIONS="-c /var/lib/tvheadend/config -u tvheadend -g video -6 -C --http_port 9981 --htsp_port 9982"
